Language.md

```diff
@@ -1 +1,74 @@
-# COMING SOON
+# JavaScript &#10084;
+A famous man once said JavaScript is the Language made for the Gods.
+Okay I said  it and I am neither famous nor very smart, but going by the stats JavaScript is *the* most famous language in the world right now.
```

Comment:
Link to SO's [survey](http://www.eweek.com/developer/javascript-most-popular-language-stack-overflow-report.html) and NPM's module [count graph](http://www.modulecounts.com/).